templates/rapidsai/Devel.dockerfile.j2

```diff
@@ -0,0 +1,33 @@
+# RAPIDS Dockerfile for {{ os }} "devel" image
+#
+# RAPIDS is built from-source and installed in the base conda environment. The
+# sources and toolchains to build RAPIDS are included in this image. RAPIDS
+# jupyter notebooks are also provided, as well as jupyterlab and all the
+# dependencies required to run them.
+#
+# Copyright (c) {{ now.year }}, NVIDIA CORPORATION.
+
+ARG CUDA_VER={{ DEFAULT_CUDA_VERSION }}
+ARG LINUX_VER={{ os }}
+ARG PYTHON_VER={{ DEFAULT_PYTHON_VERSION }}
+ARG RAPIDS_VER={{ DEFAULT_RAPIDS_VERSION }}
+ARG FROM_IMAGE=rapidsai/rapidsai-dev-nightly
+
+FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-devel-${LINUX_VER}-py${PYTHON_VER}
+
+ARG RAPIDS_VER
+
+{# Additions for BlazingSQL, change the default WORKDIR to run BlazingSQL NBs by default #}
+ENV BLAZING_DIR=/blazing
```

Comment:
Should the `blazingdb` repo be cloned into this directory or clone it to the same directory as rapids? (`/rapids`)

Reply:
another good question. I'd think the `/rapids` repo if `blazing` is going to become a first-class RAPIDS lib soon. Although I'd confirm this with Mike.

Reply:
I think `/blazing` is fine and is distinct enough